compiler/src/iree/compiler/Preprocessing/Common/ApplyPDLLPatterns.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "Utilities.h"
```

Comment:
Use the full (relative to IREE compiler source root) path for includes.

```suggestion
#include "iree/compiler/Preprocessing/Common/Utilities.h"
```